ste/mgr-JobPartMgr.go

```diff
@@ -583,8 +583,34 @@ func (jpm *jobPartMgr) resourceDstData(fullFilePath string, dataFileToXfer []byt
 	return common.ResourceHTTPHeaders{ContentType: jpm.inferContentType(fullFilePath, dataFileToXfer), ContentLanguage: jpm.httpHeaders.ContentLanguage, ContentDisposition: jpm.httpHeaders.ContentDisposition, ContentEncoding: jpm.httpHeaders.ContentEncoding, CacheControl: jpm.httpHeaders.CacheControl}, jpm.metadata
 }
 
+// TODO do we want these charset=utf-8?
```

Comment:
I agree we should probably drop the charset variables in there. It may be undesirable to a customer who opts to not use UTF-8 for one reason or another.